includes/class-newspack-newsletters.php

```diff
@@ -780,6 +799,34 @@ public static function default_title( $post_title, $post ) {
 		return $post_title;
 	}
 
+	/**
+	 * Handle custom Newsletter styling when viewing the newsletter as a public post.
+	 */
+	public static function public_newsletter_custom_style() {
```

Comment:
These styles will be included in the Newsletter archive page. This doesn't seem to have any obvious effect but it could cause unexpected results. Suggested fix:

```
if ( ! is_single() ) {
	return;
}
```

Reply:
Fixed in 9578e710708c9734db4cd8a7920180c2e899292f